components/aws/sagemaker/unit_tests/Dockerfile

```diff
@@ -0,0 +1,31 @@
+# How to build this docker file
+#
+# https://github.com/kubeflow/pipelines.git
+#
+# cd pipelines/components/aws
+#
+# docker build ./ -f ./sagemaker/unit_tests/Dockerfile -t amazon/unit-test-aws-sagemaker-kfp-components:20200427
```

Comment:
nit: tag doesn't need to contain datestamp. I would recommend `amazon/unit-test-sagemaker-components` perhaps?

Reply:
done